Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackFormController.swift

```diff
@@ -0,0 +1,204 @@
+            }
+        }
+        
+        let status = PHPhotoLibrary.authorizationStatus()
+        switch status {
+        case .notDetermined:
+            if #available(iOS 14, *) {
+                PHPhotoLibrary.requestAuthorization(for: .readWrite) {
+                    SentryLog.debug("Photos authorization level: \($0)")
+                    presentPicker()
+                }
+            } else {
+                PHPhotoLibrary.requestAuthorization {
+                    SentryLog.debug("Photos authorization level: \($0)")
+                    presentPicker()
+                }
+            }
+        default:
+            SentryLog.debug("Photos authorization level: \(status)")
+            presentPicker()
+        }
+#endif // SENTRY_TEST || SENTRY_TEST_CI
+    }
+    
+    func submitFeedback() {
+        switch viewModel.validate() {
+        case .success(_):
+            let feedback = viewModel.feedbackObject()
+            SentryLog.debug("Sending user feedback")
+            if let block = config.onSubmitSuccess {
+                block(feedback.dataDictionary())
+            }
+            delegate?.finished(with: feedback)
+        case .failure(let error):
+            func presentAlert(message: String, errorCode: Int, info: [String: Any]) {
+                let alert = UIAlertController(title: "Error", message: message, preferredStyle: .alert)
+                alert.addAction(UIAlertAction(title: "OK", style: .default))
+                present(alert, animated: config.animations) {
+                    if let block = self.config.onSubmitError {
+                        // we use NSError here instead of Swift.Error because NSError automatically bridges to Swift.Error, but the same is not true in the other direction if you want to include a userInfo dictionary. Using Swift.Error would require additional implementation for this to work with ObjC consumers.
+                        block(NSError(domain: "io.sentry.error", code: errorCode, userInfo: info))
+                    }
+                }
+            }
+            
+            guard case let SentryUserFeedbackFormViewModel.InputError.validationError(missing) = error else {
+                SentryLog.warning("Unexpected error type.")
+                presentAlert(message: "Unexpected client error.", errorCode: 2, info: [NSLocalizedDescriptionKey: "Client error: ."])
+                return
+            }
+            
+            presentAlert(message: error.description, errorCode: 1, info: ["missing_fields": missing, NSLocalizedDescriptionKey: "The user did not complete the feedback form."])
+        }
+    }
+    
+    func cancel() {
+        delegate?.finished(with: nil)
+    }
+}
+
+// MARK: UIImagePickerControllerDelegate & UINavigationControllerDelegate
+@available(iOS 13.0, *)
+extension SentryUserFeedbackFormController: UIImagePickerControllerDelegate & UINavigationControllerDelegate {
```

Comment:
`l`: Is there a difference between:

```
extension SentryUserFeedbackFormController: UIImagePickerControllerDelegate & UINavigationControllerDelegate {
```

and 

```
extension SentryUserFeedbackFormController: UIImagePickerControllerDelegate, UINavigationControllerDelegate {
```

Reply:
There doesn't appear to be, it compiles and works all the same if I change it to a comma. I simply copied and pasted this from the type of the declaration of `delegate` in UIImagePickerController
![image](https://github.com/user-attachments/assets/741d9850-0617-4f9d-ae8f-f7a02d4ff9b8)

Reply:
None that I can see. I simply copypasted the type of `delegate` from its declaration in `UIImagePickerController`